oauth2/src/main/resources/application-docker.yml

```diff
@@ -23,4 +23,11 @@ spring:
   liquibase:
     contexts: docker
     change-log: classpath:db/changelog/db.changelog-master.xml
+    url:  jdbc:postgresql://localhost:5432/userdb
```

Comment:
here should be `jdbc:postgresql://postgres:5432/userdb`

Reply:
okok